docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,382 @@
+  [helloworld] Building Android APK...
+  Starting a Gradle Daemon
+  ...
+  BUILD SUCCESSFUL in 1m 1s
+  28 actionable tasks: 17 executed, 11 up-to-date
+  [helloworld] Built android/Hello World/app/build/outputs/apk/debug/app-debug.apk
+
+Run the app on a virtual device
+===============================
+
+We're now ready to run our application. You can use Briefcase's ``run`` command
+to run the app on an Android device, either physical or virtual. This section
+covers running the app on a virtual Android device.
+
+On Android, Briefcase's ``run`` command requires that you specify which device to run on.
+If you use ``run`` without specifying a device, Briefcase will tell you how to
+create an appropriate Android virtual device. Run that command on your system.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      No Android device was specified. Please specify a specific device on which
+      to run the app by passing `-d <device_id>`.
+
+      ...
+
+      If you do not see any devices, you can create and start an emulator by running:
+
+      ...
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      No Android device was specified. Please specify a specific device on which
+      to run the app by passing `-d <device_id>`.
+
+      ...
+
+      If you do not see any devices, you can create and start an emulator by running:
+
+      ...
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>briefcase run android
+
+      No Android device was specified. Please specify a specific device on which
+      to run the app by passing `-d <device_id>`.
+
+      ...
+
+      If you do not see any devices, you can create and start an emulator by running:
+
+      ...
+
+
+Examine the specific output on your system, and run the commands provided
+until you see an Android virtual device on your screen. It will look something
+like this as it starts up.
+
+.. figure:: ../images/android/tutorial-5-booting.png
+   :alt: Android virtual device booting
+
+   Android virtual device booting
+
+.. figure:: ../images/android/tutorial-5-running.png
+   :alt: Android virtual device fully started
+
+   Android virtual device fully started
+
+Once an Android virtual device is running, you can launch the app with Briefcase
+by running this command:
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android -d emulator-5554
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android -d emulator-5554
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>briefcase run android -d emulator-5554
+
+At this point, you should see the app launch on your virtual device. If you see
+an error message, you may need to follow the advice in the error message about
+finding the correct device name.
```

Comment:
Mental note: We need a screenshot here, too.

Reply:
I'll add that to a list of blockers in the PR description, so we don't forget.